zio-http/src/main/scala/zio/http/model/headers/values/AcceptCharset.scala

```diff
@@ -0,0 +1,72 @@
+package zio.http.model.headers.values
+
+sealed trait AcceptCharset {
+  val raw: String
+}
+
+/**
+ * The Accept-Charset request HTTP header was a header that advertised a
+ * client's supported character encodings. It is no longer widely used. UTF-8 is
+ * well-supported and the overwhelmingly preferred choice for character
+ * encoding. To guarantee better privacy through less configuration-based
+ * entropy, all browsers omit the Accept-Charset header. Chrome, Firefox,
+ * Internet Explorer, Opera, and Safari abandoned this header.
+ */
+object AcceptCharset {
+
+  /**
+   * Seven-bit ASCII, a.k.a. ISO646-US, a.k.a. the Basic Latin block of the
+   * Unicode character set
+   */
+  object `US-ASCII` extends AcceptCharset {
+    override val raw: String = "US-ASCII"
+  }
+
+  /**
+   * ISO Latin Alphabet No. 1, a.k.a. ISO-LATIN-1
+   */
+  object `ISO-8859-1` extends AcceptCharset {
```

Comment:
I'm not sure about this convention. I see it in some Scala projects. But others would do `ISO_8859_1`, which I think I have a slight preference for.

Reply:
All right, let's replicate that here.